src/App.css

```diff
@@ -1,38 +1,347 @@
+
+html { box-sizing: border-box;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+/* base styles */
+
+img {
+  max-width: 100%;
+  display: block;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+
+.wrapper {
+  max-width: 700px;
+  width: 70%;
+  margin: 0 auto;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+button {
+  background: none;
+  border: none;
+  cursor: pointer;
+  font-family: 'Give You Glory', cursive;
+}
+
+html {
+  font-size: clamp(1rem, (1rem + 0.25vw), 4rem);
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+
+}
+
+h1,
+h2,
+ul,
+li,
+a,
+input,
+label,
+button,
+p {
+  margin: 0;
+  padding: 0;
+  color: #000;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  color: inherit;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+
+/* header styling */
+  
+header {
+  background: url(./assets/backgroundImage.png);
+  background-attachment: scroll;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
+}
+
+header .wrapper {
+  background-color: rgba(26, 83, 92, 0.9);
+  border-radius: 5px;
+
+}
+
+h1 {
+  font-family: 'Monoton', cursive;
+  color: #ff6b6b;
+  padding-top: 1.5rem;
+  text-align: center;
+  padding-bottom: 1rem;
+  font-size: 3rem;
+}
+
+
+h2 {
   color: white;
+  text-align: center;
+  font-size: 1.2rem;
+  padding-bottom: 1rem;
+}
+
+header p {
+  color: white;
+  font-size: 1rem;
+  padding-bottom: 0.75rem;
+  text-align: center;
+}
+
+header .landingNav {
+  margin-top: 2rem;
+  width: 100%;
+}
+
+.landingNav ul {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.landingNav a {
+  background-color: rgba(26, 83, 92, 0.9);
+  border: rgba(26, 83, 92, 0.9) 2px solid;
+  border-radius: 5px;
+  color: white;
+  font-size: 1.2rem;
+  text-decoration: none;  
+  padding: 0.5rem 1rem;
+  transition: color 0.3s ease-in-out, background-color 0.3s ease-in-out, border 0.3s ease-in-out;
+  -webkit-box-shadow: 8px 8px 0px 4px #1A535C; 
+  box-shadow: 8px 8px 0px 4px #1A535C;
+}
+
+.landingNav a:hover {
+  color: #1A535C;
+  background-color: white;
+  border: #1A535C 2px solid;
+}
+
+.landingNav a:focus {
+  color: #1A535C;   
+  background-color: white;
+}
+
+/* form styling */
+
+form {
+	margin: 1rem 0;
+	padding: 1rem 0 ;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+  border: #1A535C 2px solid;
+  border-radius: 5px;
+}
+
+form label {
```

Comment:
I think the form label can also use a bit more padding! At 320px, the text is close to the teal border